### ISSUE TITLE
Locked pdex but no open orders

### ISSUE THREAD

**Issue by jaymansfield** [Polkadex-Substrate/Polkadex-Open-Beta#414]
Locked pdex but no open orders on the beta (showing in both web app and mobile app)

trading account esmTkMQd76arMnRjWFkoEDj8pz5B4AuoTLXYDjtVixk4oNhVt
funding account esnbKSRnU1WKNgJewte4asokmCWwiKBL2HeVWS3NSYEiZfoYc

![image](https://user-images.githubusercontent.com/6218678/201357432-9be789f7-eefa-4514-9107-5a537a722b3d.png)

![image](https://user-images.githubusercontent.com/6218678/201357452-f51fb95f-30ca-4230-8442-5bc12bbb9fc8.png)

onchainaddress:{esm99RRxKvainDsRR6UHRcju4vNnAw1ouZEw64Yfn12tc6HpQ}

**Comment by nuel77** [comment]
is this locked amout still after trying a refresh?

**Comment by jaymansfield** [comment]
@nuel77 yes still locked. Can see it in both the mobile app and the web app still.

**Comment by jaymansfield** [comment]
![image](https://user-images.githubusercontent.com/6218678/201397333-a3f163c3-efb5-430c-8eb4-6f51cc83ec9e.png)
![image](https://user-images.githubusercontent.com/6218678/201397386-da8c37a8-eca8-4f5d-a0c3-23b0d3a480d3.png)
![image](https://user-images.githubusercontent.com/6218678/201397405-483c207a-d68b-4184-b42a-ab6ed64d9974.png)

**Comment by nuel77** [comment]
hmm.. we are adding a feature for users to force unlock incase these balance locks up.. do you happen to know the trades you did to create this issue. it be good if we can recreate this and do some tests.

**Comment by jaymansfield** [comment]
@nuel77 ive tried a bunch more scenarios to try to reproduce this and just figured it out.

It happens when you delete a trading account from the blockchain that has open orders.

1. Created new account under new email
2. Created trading account 1
3. Created sell pdex order
4. Created trading account 2 and set to in use
5. Deleted trading account 1
6. I'm now left with a single trading account, and locked funds i cant access since the orders disappeared

![image](https://user-images.githubusercontent.com/6218678/202861423-7de98cb2-63c0-4acb-9317-5560a4461537.png)

![image](https://user-images.githubusercontent.com/6218678/202861431-4423f2b0-b549-460b-839d-e46fd3eaf198.png)

**Comment by jaymansfield** [comment]
it basically looks like once you delete a trading account the system does not cancel it's open orders and they become unaccessible to the user at that point
